forge/routes/api/settings.js

```diff
@@ -23,14 +23,19 @@ module.exports = async function (app) {
                 response['telemetry:enabled'] = app.settings.get('telemetry:enabled')
                 response['user:signup'] = app.settings.get('user:signup')
                 response['user:reset-password'] = app.settings.get('user:reset-password')
+                response['user:tcs-required'] = app.settings.get('user:tcs-required')
+                response['user:tcs-url'] = app.settings.get('user:tcs-url')
+                response['user:reset-password'] = app.settings.get('user:reset-password')
                 response['user:team:auto-create'] = app.settings.get('user:team:auto-create')
                 response.email = app.postoffice.exportSettings(true)
             }
             reply.send(response)
         } else {
             reply.send({
                 'user:signup': app.settings.get('user:signup') && app.postoffice.enabled(),
-                'user:reset-password': app.settings.get('user:reset-password') && app.postoffice.enabled()
+                'user:reset-password': app.settings.get('user:reset-password') && app.postoffice.enabled(),
+                'user:tcs-required': app.settings.get('user:tcs-required') && app.postoffice.enabled(),
```

Comment:
Why does email need to be enabled to require T&Cs?

Reply:
> Why does email need to be enabled to require T&Cs?

Because `user:signup` needs to be enabled - which in turn hinges on email being enabled.

It is possibly a little redundant, but no harm.